Orange/widgets/evaluate/owconfusionmatrix.py

```diff
@@ -243,6 +245,21 @@ def set_results(self, results):
         else:
             self.Error.no_regression.clear()
 
+        nan_values = False
+        if results is not None:
+            assert isinstance(results, Orange.evaluation.Results)
+            if numpy.any(numpy.isnan(results.actual)) or \
+                    numpy.any(numpy.isnan(results.predicted)):
+                # Error out here (could filter them out with a warning
+                # instead).
+                nan_values = True
+                results = data = None
+
+        if nan_values:
+            self.Error.invalid_values()
+        else:
+            self.Error.invalid_values.clear()
```

Comment:
No need to change anything here, but just if you didn't know: if you prefer, you can use `self.Error.invalid_values(shown=nan_values)`, which shows or hides the messages.